tests/app/dao/notification_dao/test_notification_dao_performance_platform.py

```diff
@@ -6,28 +11,32 @@
     KEY_TYPE_TEST,
     LETTER_TYPE,
 )
-from datetime import date, datetime, timedelta
-from freezegun import freeze_time
 
 BEGINNING_OF_DAY = date(2016, 10, 18)
 END_OF_DAY = date(2016, 10, 19)
 
 
+@pytest.mark.serial
 def test_get_total_notifications_filters_on_date_within_date_range(sample_template, sample_notification):
     template = sample_template()
     sample_notification(template=template, created_at=datetime(2016, 10, 17, 23, 59, 59))
     sample_notification(template=template, created_at=BEGINNING_OF_DAY)
     sample_notification(template=template, created_at=datetime(2016, 10, 18, 23, 59, 59))
     sample_notification(template=template, created_at=END_OF_DAY)
 
+    # Requires serial or better time-boxing
     result = dao_get_total_notifications_sent_per_day_for_performance_platform(BEGINNING_OF_DAY, END_OF_DAY)
 
     assert result.messages_total == 2
 
 
+@pytest.mark.serial
```

Comment:
I am trying to understand why this was updated for the Poetry Implementation. I am not saying it should not be done if it's unrelated. I wanted to make sure I understand why it's being done if it's related to Poetry.

Reply:
Removed all of these where able. The remaining 23 in this PR have comments in the method where it is unable to be ran in parallel.